tests/python/relay/test_pass_annotation.py

```diff
@@ -538,9 +539,10 @@ def expected():
     expected_index = [2, 2, 2, 1, 1, 1, 2, 2]
     check_annotated_graph(annotated_func, expected_func)
     params = {"a": a_data, "b": b_data, "c": c_data, "d": d_data}
-    config = {"opt_level": 0}
-    config["fallback_device"] = fallback_device
-    with relay.build_config(**config):
+    fall_dev = tvm.tir.IntImm("int32", fallback_device.device_type)
+    with tvm.transform.PassContext(opt_level=0,
+                                   config={"relay.fallback_device_type":
+                                           tvm.tir.IntImm(32, fall_dev)}):
```

Comment:
do we ned the intimm? I feel that conversion from int to the intImm is automatic

Reply:
yeah, it's not necessary. Let me update.